fluentd/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ex
+
+rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
+  yum install -y --setopt=tsflags=nodocs \
+  gcc-c++ \
+  ruby \
+  ruby-devel \
+  libcurl-devel \
+  make && \
+  gem install --no-ri fluentd fluent-plugin-kubernetes_metadata_filter fluent-plugin-elasticsearch fluent-plugin-flatten-hash && \
+  mkdir -p ${THROTTLE_CONF_LOCATION} && \
+  touch ${THROTTLE_CONF_LOCATION}/settings
+
+
+cp $HOME/fluentd_es_config.conf $HOME/fluentd_es_ops_config.conf /etc/fluent/configs.d/output
```

Comment:
I suppose I could just get rid of `$HOME/fluentd_es_config.conf` and  `$HOME/fluentd_es_ops_config.conf` and just copy them to `/etc/fluent/configs.d/output` in Dockerfile.